linux_os/guide/services/ssh/ssh_server/sshd_enable_pam/rule.yml

```diff
@@ -8,8 +8,12 @@ description: |-
     PasswordAuthentication in addition to PAM account and session module processing for all
     authentication types.
 
-    To enable PAM authentication, add or correct the following line in the
-    <tt>/etc/ssh/sshd_config</tt> file:
+    To enable PAM authentication, add or correct the following line in
+    {{% if product in ['fedora', 'rhel9'] %}}
```

Comment:
Should sshd_distributed_config be used?

Reply:
Fixed in e04685101eedd853e97bf4690afc862986495384